README.md

```diff
@@ -107,6 +107,22 @@ To run Edge in **offline** mode, use the command `offline` and provide a volume
 docker run -v ./examples:/edge/data -p 3063:3063 -e BOOTSTRAP_FILE=/edge/data/features.json -e TOKENS=<your_client_token_1,your_client_token_2> unleashorg/unleash-edge:<version> offline
 ```
 
+### Client and frontend tokens in offline mode
+
+> Availability: Unleash Edge v19.4+
+
+Offline mode supports multiple [token types](https://docs.getunleash.io/reference/api-tokens-and-client-keys).
+
+For [client tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens), use:
```

Comment:
Another option:

For [client tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens), use `CLIENT_TOKENS` or `TOKENS` environment variables or `--client-tokens` or `--tokens` CLI flags.